packages/providers/providers.yaml

```
@@ -9187,6 +9187,63 @@ pandadoc-api-key:
             example: 3039ba033eb1410caa0a2227158d63c9d6502cd8
             pattern: '^[a-fA-F0-9]+$'
 
+passportal:
+    display_name: Passportal
+    categories:
+        - support
+    auth_mode: TWO_STEP
+    token_url: https://${connectionConfig.base_url}/api/v2/auth/client_token
+    token_headers:
+        x-key: ${credentials.apiKey}
+        x-hash: ${credentials.hmacToken}
+        content-type: application/json
+    token_params:
+        scope: docs_api
+        content: ${credentials.content}
+    token_response:
+        token: access_token
+        token_expiration: data.expiry_time
+        token_expiration_strategy: expireAt
+    proxy:
+        base_url: https://${connectionConfig.base_url}
+        headers:
+            authorization: Bearer ${accessToken}
+        verification:
+            method: GET
+            headers:
+                content-type: application/json
+            endpoints:
+                - /api/v2/documents?resultsPerPage=1
+    connection_config:
+        base_url:
+            type: string
+            title: Passportal Base URL
+            description: Your Passportal base URL name (e.g., if your dashboard URL is https://mycompany.mypasswordapp.com/dashboard, use 'mycompany.mypasswordapp.com')
```

```suggestion
            description: Your Passportal base URL
```
Try to keep it short here, to be explained in details within the `connect-ui` docs. Same goes to the other fields below.